moped-database/migrations/1730931541884_reorg_funding_sources/up.sql

```diff
@@ -0,0 +1,33 @@
+-- Insert new funding source and soft-delete existing funding sources that are merging into the new one
+INSERT INTO "public"."moped_fund_sources" ("funding_source_name") VALUES
+('Austin Transportation and Public Works');
+
+UPDATE moped_fund_sources SET is_deleted = true WHERE funding_source_name IN ('Austin Transportation', 'Public Works');
+
+-- Find existing project funding records that are associated with the funding sources that are merging
+-- and update them to the new funding source called Austin Transportation and Public Works
+WITH funding_source_todos AS (
```

Comment:
I'm not entirely convinced these CTEs make this more readable since there are still sub-queries in the UPDATE statement below. Open to updating if anyone feels strongly about it!

Reply:
i like them! i wish i could bookmark this migration for the next time we need to do this kind of operation.

i suppose i can bookmark it... 🤔

Reply:
cool, and, yes, this builds off some of the fancy SQL you've written in the past that I bookmarked in my browser. Maybe a SQL recipes folder with examples like this would be handy! 🧑‍🍳 I often find myself searching through old migrations for some copypasta.